target/linux/ath79/dts/qca9558_sophos_ap55c.dts

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9558_sophos_apxx.dtsi"
```

Comment:
I'd prefer qca9558_sophos_ap.dtsi, since xx implies a two-digit number, which does not cover the AP100.

Reply:
OK, done - they have a newer line of devices named the APX series & I hope to add support for those in future, so this will be less confusing if/when that happens as well.